src/cargo/util/interning.rs

```diff
@@ -13,7 +13,19 @@ use std::str;
 use std::sync::Mutex;
 use std::sync::OnceLock;
 
-static STRING_CACHE: OnceLock<Mutex<HashSet<&'static str>>> = OnceLock::new();
+pub static DEFAULT_INTERNED_STRING: InternedString = InternedString { inner: "default" };
+
+fn cache_init() -> std::sync::MutexGuard<'static, HashSet<&'static str>> {
```

Comment:
imo this should just be called `interned_storage` or something.  The initialization is an implementation detail.